package.json

```diff
@@ -29,8 +29,7 @@
     "clean:deps": "rm -rf node_modules/.yarn-integrity && yarn install",
     "clean:dist": "rm -rf dist",
     "clean": "yarn clean:deps && yarn clean:dist",
-    "prepare": "husky install",
-    "postinstall": "yarn yarn-deduplicate yarn.lock"
```

Comment:
when I was testing I needed not delete, but update to:

postinstall: "yarn dedupe"

Reply:
did you see this @j8seangel ?

Reply:
@fertandil87 yarn 3 is doing this natively so we don't need to run the script anymore